orttraining/orttraining/test/python/_test_commons.py

```diff
@@ -1,4 +1,47 @@
 import math
+import os
+import sys
+import subprocess
+
+def is_windows():
+    return sys.platform.startswith("win")
+
+def run_subprocess(args, cwd=None, capture=False, dll_path=None,
+                   shell=False, env={}, log=None):
+    log.info("Running subprocess in '{0}'\n{1}".format(
+        cwd or os.getcwd(), args))
+    my_env = os.environ.copy()
+    if dll_path:
+        if is_windows():
+            my_env["PATH"] = dll_path + os.pathsep + my_env["PATH"]
+        else:
+            if "LD_LIBRARY_PATH" in my_env:
+                my_env["LD_LIBRARY_PATH"] += os.pathsep + dll_path
+            else:
+                my_env["LD_LIBRARY_PATH"] = dll_path
+
+    stdout, stderr = (subprocess.PIPE, subprocess.STDOUT) if capture else (
+        None, None)
+    my_env.update(env)
+    completed_process = subprocess.run(
+        args, cwd=cwd, check=True, stdout=stdout, stderr=stderr,
+        env=my_env, shell=shell)
+    log.debug("Subprocess completed. Return code=" +
+              str(completed_process.returncode))
+    return completed_process
+
+
+def run_subprocess(args, cwd=None, env={}, log=None):
```

Comment:
why do we need two run_subprocess() functions?

Reply:
My mistake. I was using the shorter one then decided to use the full method from build.py. Forgot to remove the shorter one